Code/src/nlp/trie_funcs.pyx

```diff
@@ -0,0 +1,135 @@
+import marisa_trie as marisa
+import pandas as pd
+import numpy as np
+import codecs
+import pickle
+import sys
+
+
+# TODO: Need to rewrite the file read. Right now it assumes that only one text file is used.
+# TODO: The easiest way is to probably change load_data to combine 2 and 3-grams(and maybe more)
+
+
+def autocomplete(word, data_path, triee=None):
```

Comment:
a bit hard to follow logic here; variable names could be more descriptive or could add comments (former preferred as much as possible)